internal/proto/local/controller/ops/services/v1/health_service.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package controller.ops.services.v1;
+
+option go_package = "github.com/hashicorp/boundary/internal/gen/controller/ops/services;services";
+
+import "google/api/annotations.proto";
+import "controller/ops/resources/health/v1/health.proto";
+
+service HealthService {
+  rpc GetHealth(GetHealthRequest) returns (GetHealthResponse) {
+    option (google.api.http) = {
+      get : "/health"
+    };
+  }
```

Comment:
If we wanted to, we could add some extra metadata about expected responses to our generated swagger file by adding annotations here. See https://github.com/grpc-ecosystem/grpc-gateway/blob/c20d40b67e395dbd6f8810d462c6909bcfca4fa4/examples/internal/proto/examplepb/a_bit_of_everything.proto#L593-L604 for an example. I assume we either return 200 or 503? I don't know if that sort of thing is something we'd be interested in but I thought it worth mentioning 😁.

Reply:
I played with this but couldn't get swagger to work properly - It kept returning a broken definition for the second `response` object - I'll try this example 👍

Reply:
Added in [db3bd22](https://github.com/hashicorp/boundary/pull/1882/commits/db3bd222f93a84977c791a4b2b86c1c39e9e3087) but it looks like our structure would add the swagger information [here](https://www.boundaryproject.io/api-docs/) which isn't ideal because health isn't part of the API.

Reply:
If we don't want it to be part of the `controller.swagger.json` we can generate it to a separate file, or maybe we don't need to generate this swagger documentation at all?